src/Farmer/Builders/Builders.BingSearch.fs

```diff
@@ -0,0 +1,50 @@
+[<AutoOpen>]
+module Farmer.Builders.BingSearch
+
+open Farmer
+open Farmer.Arm.BingSearch
+open Farmer.BingSearch
+
+type BingSearch =
+    /// Gets an ARM Expression key for any Bing Search instance.
+    static member getKey (resourceId: ResourceId) =
+        ArmExpression.create(sprintf "listKeys(%s, '%s').key1" resourceId.ArmExpression.Value accounts.ApiVersion, resourceId)
+    static member getKey (name: ResourceName) = BingSearch.getKey (accounts.resourceId name)
+
+type BingSearchConfig =
+    { Name: ResourceName
+      Sku: Sku
+      Tags: Map<string,string>
+      StatisticsEnabled: bool }
+    /// Gets an ARM expression to the key of this Bing Search instance.
+    member this.Key = BingSearch.getKey (accounts.resourceId this.Name)
+    interface IBuilder with
+        member this.ResourceId = accounts.resourceId this.Name
+        member this.BuildResources location = [
+            { Name = this.Name
+              Location = location
+              Sku = this.Sku
+              Tags = this.Tags
+              Properties = {| statisticsEnabled = this.StatisticsEnabled |} }
+        ]
+
+type BingSearchBuilder () =
+    member _.Yield _ =
+        { Name = ResourceName.Empty
+          Sku = F1
+          Tags = Map.empty
+          StatisticsEnabled = false }
+    [<CustomOperation "name">]
+    member _.Name (state:BingSearchConfig, name) = { state with Name = ResourceName name }
+    [<CustomOperation "sku">]
+    member _.Sku (state:BingSearchConfig, sku) = { state with Sku = sku }
+    [<CustomOperation "enable_statistics">]
+    member _.EnableStatistics (state:BingSearchConfig) = { state with StatisticsEnabled = true }
```

Comment:
Consider passing in a boolean for this - this enables better programmatic access.

Reply:
I pass a `FeatureFlag` so we would have:

```fsharp
bingSearch {
  name "test"
  statistics Enabled
}
```